src/pages/CropSidebar/CropSidebar.jsx

```diff
@@ -353,6 +367,38 @@ const CropSidebar = ({
                   )}
         />
       </SoloFilter>
+      <ListItem
+        component="div"
+      >
+        <Tooltip
+          enterTouchDelay={0}
+          title={(
+            <p>Use the Cover Crop Group Filter to select specific cover crop groups to filter by.</p>
+        )}
+        >
+          <span>
+            Cover Crop Group Filter
+            <HelpOutlineIcon style={{ cursor: 'pointer', transform: 'scale(0.7)' }} />
+          </span>
+        </Tooltip>
+      </ListItem>
+      <ListItem>
+        {coverCropGroup.map((val) => {
+          const selected = cropGroupFilterRedux === val.label;
+          return (
+            <Grid key={val.label} item>
+              <Chip
+                key={val.label}
+                onClick={() => handleCropGroupFilter(val.label)}
+                component="li"
+                size="medium"
+                label={val.label}
+                color={selected ? 'primary' : 'secondary'}
+              />
```

Comment:
Can you add some gap between the chips, just like other chip filters?
![image](https://github.com/user-attachments/assets/7a2960fc-6fd3-45ea-ba63-8863e9df3375)